torch/ao/quantization/observer.py

```diff
@@ -334,7 +335,12 @@ def reset_min_max_vals(self):
         raise NotImplementedError("Cannot reset min/max values in the given observer.")
 
 
-class MinMaxObserver(_ObserverBase):
+# Originally, this class was called `_ObserverBase`.  Keeping the old name around
+# for backwards compatibility.
```

Comment:
should we add a TODO to remove it after certain release like 1.13?

Reply:
sure